modular_sand/code/modules/research/designs/misc_designs.dm

```diff
@@ -6,7 +6,7 @@
 	materials = list(/datum/material/glass = 2250, /datum/material/plasma = 2250, /datum/material/diamond = 185, /datum/material/bluespace = 185)
 	build_path = /obj/item/reagent_containers/spray/bluespace
 	category = list("Misc", "Medical Designs")
-	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_SERVICE
```

Comment:
Do you think you can modularize this in some way? maybe by editing the flags in a modular file directly or adding that flag on `New()` ?